examples/invalidations_blog.jl

```diff
@@ -1,5 +1,5 @@
 using SnoopCompile
-using SnoopCompile: countchildren
+import SnoopCompile: countchildren
```

Comment:
If I'm *extending* a method I use `import`. If I'm just *calling* a method I use `using`, as a signal that there are no additional method of `countchildren` being defined here. But perhaps I'm missing a deeper reason for this change?

Reply:
Oh sorry. That is propagated from my closed #90 PR